recipes/salvus_mesher_lite/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "salvus_mesher_lite" %}
+{% set version = "1.0.6" %}
+{% set sha256 = "7845275d1bd627f6358c586c44feb479e973ea3e6aea808ed65f373c827e3ac8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - numpy
+    - scipy
+    - matplotlib
+    - jsonschema
+    - pyyaml
+    - pyexodus
+
+test:
+  imports:
+    - salvus_mesher_lite
+    - salvus_mesher_lite.tests
+  requires:
+    - pytest
+    - flake8
+    - pytest-mpl
+    - mock  # [py27]
+
+about:
+  home: https://salvus.io
+  license: GPL-3.0
+  summary: 'Salvus mesher for AxiSEM meshes'
+
+extra:
+  recipe-maintainers:
+    - krischer
+    - martinvandriel
```

Comment:
@martinvandriel are you okay with being a maintainer on the feedstock?

Reply:
sure!